clusterloader2/README.md

```diff
@@ -1,21 +1,81 @@
 # ClusterLoader
 
-### Running ClusterLoader
+## Running ClusterLoader
 
 To run ClusterLoader type:
 ```
-go run cmd/clusterloader.go --kubeconfig=kubeConfig.yaml --testconfig=config.json
+go run cmd/clusterloader.go --kubeconfig=kubeConfig.yaml --testconfig=config.yaml
 ```
-Flags kubeconfig and testconfig are necessary.
+Flags kubeconfig, testconfig and nodes are necessary.
 
-#### Flags
+### Flags
 
+#### Required
+
+These flags are required for any test to be run.
  - kubeconfig - path to the kubeconfig file.
- - testconfig - path to the test config file.
+ - testconfig - path to the test config file. This flag can be used multiple times
```

Comment:
Is the flag used multiple times, or it's value contains list of tests?

Reply:
Flag is used multiple times.